generator/generator.go

```diff
@@ -35,7 +35,28 @@ type TemplateInputs struct {
 	// Interface information for template
 	Interface TemplateInputInterface
 	// Vars additional vars to pass to the template, see Options.Vars
-	Vars map[string]interface{}
+	Vars    map[string]interface{}
+	Imports []string
+}
+
+// RenderImports outputs a list of imports, combining the ones from
+// the source file with any provided for the template itself.
+func (t TemplateInputs) RenderImports(imports ...string) string {
+	allImports := make(map[string]struct{})
+	for _, i := range t.Imports {
+		allImports[strings.TrimSpace(i)] = struct{}{}
+	}
+	for _, i := range imports {
+		if i[len(i)-1] != '"' {
+			i = `"` + i + `"`
+		}
+		allImports[strings.TrimSpace(i)] = struct{}{}
```

Comment:
@recht shouldn't we trim space from the user input first and then add quotes?
If you can fix this I'm ok to fix templates myself using this new helper.